.pre-commit-config.yaml

```diff
@@ -15,7 +15,7 @@ repos:
       - id: mixed-line-ending
       - id: check-added-large-files
   - repo: https://github.com/jumanjihouse/pre-commit-hooks
-    rev: 2.1.4
+    rev: 2.1.5
```

Comment:
This change doesn't belong in this PR - Maybe submit it as a separate PR for consideration.

Reply:
will do